sources/api/certdog/README.md

```diff
@@ -8,4 +8,4 @@ Current version: 0.1.0
 
 ## Colophon
 
-This text was generated using [cargo-readme](https://crates.io/crates/cargo-readme), and includes the rustdoc from `src/main.rs`.
\ No newline at end of file
+This text was generated using [cargo-readme](https://crates.io/crates/cargo-readme), and includes the rustdoc from `src/main.rs`.
```

Comment:
I'm wondering what this change does? It seems like it makes no difference in the rich diff. The reason I ask is because all the READMEs that have this are auto-generated from docstrings from our source code, so I worry that this change won't last when we touch the source in any way.

Reply:
I'm wondering the same thing: I'm not seeing a change in the GitHub UI and it doesn't appear to be adding a newline at the end?

Reply:
Yes, it is the newline at EOF. It was missing in many of these files. It is strange the GitHub is not rendering the symbol that it renders in some views.

It's automatically added when I run `cargo readme > README.md`. I see that there's a `generate-readme` package that is supposed to automatically generate these but I don't know if there's an easy way to invoke this. I suppose it happens at build time. I just built this and there were no changes to the README files.

Here's an alternative way to view the diff: https://github.com/bottlerocket-os/bottlerocket/commit/d5487d348f6bbf1e8c2ea973d0f23441d37d88fc.diff

```diff

diff --git a/sources/api/certdog/README.md b/sources/api/certdog/README.md
index 8769328a9c..e87b8418e0 100644
--- a/sources/api/certdog/README.md
+++ b/sources/api/certdog/README.md
@@ -8,4 +8,4 @@ Current version: 0.1.0
 
 ## Colophon
 
-This text was generated using [cargo-readme](https://crates.io/crates/cargo-readme), and includes the rustdoc from `src/main.rs`.
\ No newline at end of file
+This text was generated using [cargo-readme](https://crates.io/crates/cargo-readme), and includes the rustdoc from `src/main.rs`.
```